src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -159,11 +159,9 @@ public List<PhotonPipelineResult> getResultsBackCam() {
     }
 
     /**
-     * Returns the latest standard deviations of the estimated pose from {@link
-     * #getEstimatedGlobalPose()}, for use with {@link
-     * edu.wpi.first.math.estimator.SwerveDrivePoseEstimator
-     * SwerveDrivePoseEstimator}. This should
-     * only be used when there are targets visible.
+     * Returns the latest standard deviations of the estimated pose from {@link #frontCamGetEstimatedGlobalPose} and {@link #backCamGetEstimatedGlobalPose}.
```

Comment:
I feel like this is going to conflict with the auto-formatting from the Java extension.